cmd/zstdseek/main.go

```diff
@@ -60,13 +62,27 @@ func main() {
 		logger.Fatal("verify can't be used with stdout output")
 	}
 
+	bar := progressbar.DefaultSilent(0, "")
+
 	var input io.ReadCloser
 	if inputFlag == "-" {
 		input = os.Stdin
 	} else {
-		if input, err = os.Open(inputFlag); err != nil {
+		file, err := os.Open(inputFlag)
```

Comment:
nit: you do not need a separate `file` variable, you can use `input`.

Reply:
The type for `input` is a `io.ReadCloser`, not the file. I can just use `os.Stat`, too.